src/views/Layout.css

```diff
@@ -7,18 +7,8 @@
  *
  * @see: https://developer.mozilla.org/en-US/docs/Web/CSS/env
  */
```

Comment:
I think maybe we can delete this file and Home.css

Reply:
I would like to keep that for the cleaning we will do once we are done with the styling.

Reply:
I'd rather do the cleaning as soon as you don't need it!